### PR TITLE
Fix missing fmt argument in error message

### PR DESCRIPTION
**Pull request by dtolnay**
## Motivation

Without this, the error message would literally be "invalid length for u32: {}" with curly braces in it instead of the input len.

## Test Plan

`cd storage`
`cargo check`

### CODE REVIEW

**Comment by niklaslong** [comment]
Thank you for catching and fixing this!

**Review by niklaslong — APPROVED** [review]
Noncritical, let's wait until the testnet 2 changes are in to merge.

**Comment by ljedrz** [comment]
Closing the PR as it's either no longer applicable to testnet2 (I checked both snarkOS and snarkVM).

**Pull request closed without merging**
